### DISCUSSION TITLE
Integration of Standard Data Structures/Collections in GDScript

### DISCUSSION THREAD

**Discussion by JNatheriver** [godotengine/godot-proposals#8971] [Scripting]
I propose the integration of standard data structures and collections directly into GDScript to enhance its functionality and provide a more robust programming experience for developers. Currently, GDScript lacks built-in support for fundamental data structures such as queues, stacks, and linked lists, sets, leading developers to reinvent the wheel by implementing these structures manually.

**Productivity Boost:**
Standard data structures are fundamental tools for software development. Integrating them directly into GDScript will reduce development time, allowing developers to focus on higher-level logic and creativity.

**Reduced Boilerplate Code:**
With built-in data structures, developers can avoid writing repetitive and error-prone code for implementing common structures, leading to cleaner and more maintainable codebases.

**Comment by iltenahmet** [comment]
I agree, I think this would be very useful.

**Comment by AThousandShips** [comment]
@iltenahmet Please don't bump without contributing significant new information. Use the :+1: reaction button on the first post instead.

**Reply by iltenahmet** [reply]
Oh, i see. Thanks for pointing this out.
